metadata-ws/src/test/resources/features/publication.feature

```diff
@@ -0,0 +1,32 @@
+Feature: publication object
+
+  Scenario: register a publication successfully
+    When I request POST /publications with JSON payload:
+    """
+    {
+      "publicationId": "Publication1"
+    }
+    """
+    Then set the URL to PUBLICATION
+    And the response code should be 201
+    And the Location header should be present with value of PUBLICATION
+    When I request GET with value of PUBLICATION
+    Then the response code should be 200
+    And the response JSON should be:
+    """
+    {
+      "publicationId": "Publication1"
```

Comment:
why is id and lastModifieddate not visible in json ?

Reply:
Its part of object. Both are auto generated and we dont explicitly include (same for other objects).
Let me know, i can include to verify id as number and lastModifieddate exists

Reply:
if its a part of the object it should be visible. and also the ticket is adding auto-id, so it should be visible.